exporter/clickhouseexporter/config.go

```diff
@@ -50,6 +52,21 @@ type Config struct {
 	// Ignored if async inserts are configured in the `endpoint` or `connection_params`.
 	// Async inserts may still be overridden server-side.
 	AsyncInsert bool `mapstructure:"async_insert"`
+	// MetricsTables defines the table names for metric types.
+	MetricsTables MetricTablesConfig `mapstructure:"metrics_tables"`
+}
+
+type MetricTablesConfig struct {
```

Comment:
Much better name 👌

Reply:
we need also mark `MetricsTableName` will be deprecated in  the `Config`

Reply:
I am not familiar with the standard deprecation process. Is there any documentation available? If so, we should adhere to the standard process.

Reply:
According to [this comment](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34251#discussion_r1702340240) the current state should be enough